src/operations/expansions/expandShell.js

```diff
@@ -0,0 +1,215 @@
+const { EPS } = require('../../math/constants')
+
+const { mat4, vec3 } = require('../../math')
+
+const { geom3, poly3 } = require('../../geometry')
+
+const { sphere } = require('../../primitives')
+
+const retessellate = require('../booleans/retessellate')
+const unionGeom3Sub = require('../booleans/unionGeom3Sub')
+
+const extrudePolygon = require('./extrudePolygon')
+
+const fnNumberSort = (a, b) => {
```

Comment:
don't we have that in the utils ?

Reply:
It will be when the PR for performance is merged. :)

Reply:
Done